### PR TITLE
feat: [OV-43587] Added char limit

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: packages/Input/src/input.scss [hunk 1]
```diff
@@ -33,6 +33,15 @@
   --ids-input-width-suffix: #{tokens.$space-7};
   --ids-input-width-suffix-compact: #{tokens.$space-6};
   --ids-input-padding-suffix: #{tokens.$space-2} #{tokens.$space-3} #{tokens.$space-2} #{tokens.$space-2};
+  --ids-input-padding-suffix-with-indicator: #{tokens.$space-2} #{tokens.$space-2} #{tokens.$space-2} #{tokens.$space-2};
```
Comment (ofrogon) [inline packages/Input/src/input.scss:36]: `--ids-input-padding-suffix-with-indicator` is unused.

File: packages/Input/src/Input.tsx [hunk 2]
```diff
@@ -143,11 +168,25 @@ const Input: React.FunctionComponent<InputProps> = React.forwardRef(
...
+          <div
+            className={cx('ids-input__asset-wrapper', {
+              'ids-input__asset-wrapper--has-char-indicator':
+                displayCharIndicator,
+            })}
+          >
+            {suffixIcon && addSuffixIcon}
+            {displayCharIndicator && (
+              <div className="ids-input__char-indicator">
+                {inputMaxLength - charLength}
```
Comment (ofrogon) [inline packages/Input/src/Input.tsx:185]: Their is something with the character count that act strange. For some reason the length of `😀😀` is the same as `😀😀a`

Visually:
image
image

Comment (vicky-comeau) [reply packages/Input/src/Input.tsx:185]: It seems like the original fix we had for emojis is no longer needed. It was now causing issues.

**Comment by franckgaudin** [comment]
The issue with emojis is unfortunately still with us. 
If we add an emoji when there is 1 character left, the remaining number will be -1.

input-emoji

**Review by franckgaudin — APPROVED** [review]

**Review by ofrogon — APPROVED** [review]

**Pull request merged**